src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -304,6 +321,10 @@ public void validateDar(User user, DataAccessRequest dar) {
     userService.hasValidActiveERACredentials(user);
```

Comment:
Not your code, but I was a little thrown by the name of this `void` method. It throws if the user doesn't have valid credentials. Barring renaming the method to something like `validateActiveERACredentials` (definitely outside the scope of this PR), maybe we can add a comment here for clarity.

Reply:
Easy peasy to rename.  Done.